client/components/notice/style.scss

```diff
@@ -57,15 +57,15 @@
 		}
 	}
 
-	.notice__dismiss {
+	.notice__wcs-dismiss {
```

Comment:
Using a custom WCS&T dismiss class to overwrite the position and colour of the dismiss button. Reason being is because WCS&T loads a few other custom styles which also include `.notice__dismiss`. For example,
![image](https://user-images.githubusercontent.com/572862/204385098-0c74bf45-861e-4bc5-83e3-a157072c0ace.png)


In the above, we can see that the `.global-notices` and the media query overwrites the location of the `X`. We could modify padding inside the global-notices classes but that could undo changes like [this](https://github.com/Automattic/woocommerce-services/pull/2130).

Since other plugin or WCS&T itself can have the same Calypso's notice component style, WCS&T can have its own padding overwrite.